app/Http/Controllers/QuestionController.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Validator;
+use App\Question;
+use App\QuestionOption;
+use Illuminate\Http\Request;
+use App\Http\Resources\Question as QuestionResource;
+
+class QuestionController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $questions = Question::all();
+
+        return QuestionResource::collection($questions);
```

Comment:
Add a QuestionShortResource, which will not contain the options and will show a return a question with a limited length of 50 symbols.

Reply:
It's non-reasonable to return all the information, including the relations with the list resource.